content/sensu-go/6.3/observability-pipeline/observe-entities/_index.md

```diff
@@ -177,6 +177,74 @@ Proxy entity registration differs from keepalive-based registration because the
 
 See [Monitor external resources][1] to learn how to use a proxy entity to monitor a website.
 
+## Service entities
+
+A service entity represents a business service in [business service monitoring (BSM)][8].
+Sensu processes service entity events just like events generated for agent and proxy entities.
+You can also use service entities for proxy check requests and events.
+
+This example shows a service entity resource definition:
+
+{{< language-toggle >}}
+
+{{< code yml >}}
+---
+type: Entity
+api_version: core/v2
+metadata:
+  created_by: admin
+  name: postgresql
+  namespace: default
+spec:
+  deregister: false
+  deregistration: {}
```

Comment:
Let's remove these values from the example as they hold no meaning for service entities.

Ditto for sensu_agent_version, system, last_seen, and subscriptions.